targets/LumberyardSdk/templates/Code/Source/PlayFabAPI.cpp.ejs

```diff
@@ -5,17 +5,22 @@
 using namespace PlayFab<%- gemName %>Sdk;
 <% if(hasClientOptions) { %>
 // Client-Specific
-AZStd::string* PlayFabClientApi::mUserSessionTicket = nullptr;
+AZStd::string PlayFabClientApi::mUserSessionTicket;
 
 bool PlayFabClientApi::IsClientLoggedIn()
 {
-    return mUserSessionTicket != nullptr && mUserSessionTicket->length() != 0;
+    return mUserSessionTicket.length() != 0;
 }
 
 // #THIRD_KIND_PLAYFAB_SHUTDOWN_FIXES: - Added a logout function so that the statics can be destroyed before the system allocator.
 void PlayFabClientApi::ForgetClientCredentials()
 {
-    SAFE_DELETE(PlayFabClientApi::mUserSessionTicket);
+    // #THIRD_KIND_PLAYFAB_CRASH_WHEN_PC_IS_OFFLINE: szaluga (2017/08/05)
```

Comment:
Aha, I finally see it.
OK, I'll have to double check this one, because the conversion was scattered in various pieces, but I can finally see someplace where it seems that the change was deliberate.